Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1168,6 +1178,114 @@ function Set-IdentityUri {
     }
 }
 
+function Get-WebDeployPackagePath{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $standalonePath,
+        [Parameter(Mandatory=$true)]
+        [string] $installerPath
+    )
+    if(Test-Path $standalonePath){
+        return Resolve-Path $standalonePath
+    }
+    if(Test-Path $installerPath){
+        return Resolve-Path $installerPath
+    }
+    Write-DosMessage -Level "Fatal" -Message "Could not find the web deploy package at $standalonePath or $installerPath."
+}
+
+function Get-WebDeployParameters{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $serviceConfig,
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $commonConfig
+    )
+
+    Confirm-ServiceConfig -commonConfig $commonConfig -serviceConfig $serviceConfig 
+    $metaDataConnectionString = Get-MetadataConnectionString -commonConfig $commonConfig -serviceConfig $serviceConfig
+    
+    $webDeployParameters = @(
+                                @{
+                                    Name = "IIS Web Application Name";
+                                    Value = "$($serviceConfig.siteName)/$($serviceConfig.appName)"
+                                },
+                                @{
+                                    Name = "Application Endpoint Address";
+                                    Value = "https://$($commonConfig.webServerDomain)/$($serviceConfig.appName)"
+                                },
+                                @{
+                                    Name =  "MetadataContext-Deployment-Deployment Connection String";
+                                    Value = $metaDataConnectionString
+                                },
+                                @{
+                                    Name = "MetadataContext-Web.config Connection String";
+                                    Value = $metaDataConnectionString
+                                }
+                            )
+    
+    return $webDeployParameters
+}
+
+function Confirm-ServiceConfig{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $commonConfig,
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $serviceConfig
+    )
+
+    Confirm-SettingIsNotNull -settingName "common.sqlServerAddress" -settingValue $commonConfig.sqlServerAddress
+    Confirm-SettingIsNotNull -settingName "common.metadataDbName" -settingValue $commonConfig.metadataDbName
+    Confirm-SettingIsNotNull -settingName "common.webServerDomain" -settingValue $commonConfig.webServerDomain
+    Confirm-SettingIsNotNull -settingName "common.clientEnvironment" -settingValue $commonConfig.clientEnvironment
```

Comment:
it doesn't look like we're using this, I would say to not include these possibly, but I'm not sure what else is dependent on the confirm-serviceconfig

Reply:
They are used in WebDeployParameters